veux.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Prateek Bhatia (ꜰʀᴇᴀᴋʏ ᴘʀᴀᴛᴇᴇᴋ)",
+			"oem": "Redmi/Poco",
+			"device": "veux/peux",
+			"filename": "",
```

Comment:
missing data
filename, download, timestamp, etc
use generated json file and adapt accordingly, don't create one manually